internal/controller/worker_controller_test.go

```diff
@@ -29,152 +39,413 @@ var (
 			},
 		},
 	}
+
+	testNamespace = "ns"
```

Comment:
Nit: it's helpful to disambiguate between temporal and k8s namespaces throughout this codebase 😄 

```suggestion
	testTemporalNamespace = "ns"
```

Reply:
totally agree! thanks for bringing this to my attention!